CAPSTONE PROJECT

Josphat205 opened the pull request:
### THIS IS HOW I DEVELOPED MY **PROJECT**

- Developed a school website **Elite school website**
- Website has two pages home and about page
- Added the responsiveness mobile and desktop version
- Added a professional readme file
- Added a video describing the project. 
- Click here to see the video https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2

prolajumokeoni commented on README.md:
```diff
@@ -1 +1,59 @@
+## Link to the Video
+[![Video](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)
+
+## **Acknowledgements**
```
- [ ] In your  README  kindly  give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

prolajumokeoni commented on README.md:
```diff
@@ -1 +1,59 @@
+![Desktop Version](img/mob4.png)
+## Usage
+
+---
```
- [ ] Kindly add a live demo section containing deployed link of your portfolio, use GitHub Pages to deploy the web pages and all pages, links, and images work as expected.

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,213 @@
+
+<body>
+  <header>
+    <nav>
```
- [ ] Kindly add the top dark nav to your header as this is part of the project requirements 👍🏿 

![Screenshot_206](https://user-images.githubusercontent.com/69638013/169273428-078c1434-db20-43e6-b126-d282fd5b4923.png)

prolajumokeoni commented on about.html:
```diff
@@ -0,0 +1,136 @@
+</head>
+
+<body>
+  <header>
```
- [ ] Kindly add the top dark nav to your header as this is part of the project requirements 👍🏿 

![Screenshot_206](https://user-images.githubusercontent.com/69638013/169273428-078c1434-db20-43e6-b126-d282fd5b4923.png)

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,213 @@
+    <nav>
+      <div class="logo">
+        <a href="#home">
+          <img src="img/logo.png" alt="" class="logo">
```
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,213 @@
+      </div>
+    </div>
+    <section class="home-pg" id="about">
+      <img class=" circle r1" src="img/round.png" alt="">
+      <img class=" circle r4" src="img/r4.png" alt="">
```
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible

prolajumokeoni commented on js/main.js:
```diff
@@ -0,0 +1,121 @@
+for (let i = 0; i < staff.length; i += 1) {
+  demo.innerHTML += `<div class="toggle"><div class="team">
+  <div class="image-div">
+    <img src="img/rec.jpg" alt="" class="pattern">
```
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible

prolajumokeoni commented on about.html:
```diff
@@ -0,0 +1,136 @@
+    <nav>
+      <div class="logo">
+        <a href="#home">
+          <img src="img/logo.png" alt="" class="logo">
```
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible

prolajumokeoni commented on js/main.js:
```diff
@@ -0,0 +1,121 @@
+  `;
+}
+
+$('#see-me').on('click', () => {
+  $('.toggle:hidden').slice(0, 2).slideDown();
+  if ($('.toggle:hidden').length === 0) {
+    $('#see-me').fadeOut();
+  }
+});
```
- Good job implementing this functionality, this is cool 😎

prolajumokeoni reviewed (changes requested):
Hi @Josphat205, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] Beautiful presentation 👏🏿 please highlight some interesting piece of code or something you built that you are very proud of in your next presentation,  please note that the maximum length of your video is 5 minutes 😉 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Josphat205 commented:
### This is the changes that I made

- Added an alternative text to images
- Added the darker layer on top of the project
- Added an acknowledgment to Cindy shin
- Added a live demo to my Readme file

zhadier reviewed (changes requested):
# Hi @Josphat205 👋 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

<img src="https://media4.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

zhadier commented on README.md:
```diff
@@ -47,7 +47,7 @@ Options:
 
 ## Link to the Video
-[![Video](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)
+- Click here to watch the Video [![Video](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)
```
- [ ] Hey @Josphat205 , great job implementing all of the required changes, I love your design and the animation effects are awesome. But as the previous Reviewer said, your video should not exceed 5 minutes. This is good practice for interviews where you have to explain your point of view on the matter in a limited amount of time (usually 3 minutes) so please make sure your video does not exceed 5 minutes.

zhadier reviewed (changes requested):
# Hi @Josphat205 👋 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

<img src="https://media4.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Josphat205 commented:
### Changes made 

- Added 4 minute video as recommended

Josphat205 commented:
### Changes Made

- Added a shorter video as recommended
- Click here to view the video https://www.loom.com/share/298c65115ace4836aa7fc2d43527724b

omar-muhamad commented on css/style.css:
```diff
@@ -0,0 +1,1172 @@
+  height: auto;
+}
+
+.wrapper {
+  width: 100%;
+  box-shadow: 0 3px 10px rgba(0, 0, 0, 0.2);
+  height: auto;
+  margin: 0 auto;
+  padding: 20px 15px;
+  background-color: #f3f3fa;
+}
```
- [ ] [Optional] [Highly recommended] In the mobile view on the Home page there is a right margin appears on the right side of the page that makes the design not look good or responsive. So please fix that.

![image](https://user-images.githubusercontent.com/68796232/169310172-ba1bd651-5255-4f64-84d4-9a13eee5c640.png)

[Notice] You may not see it because of the browser cash if you want to see it just go to the developer tools and select any mobile phone view then switch the mobile view on and off twice to see it (see the image below).

![image](https://user-images.githubusercontent.com/68796232/169311184-8e668859-2b69-4a00-9934-548ffbbc9a2c.png)

omar-muhamad commented on css/style.css:
```diff
@@ -0,0 +1,1172 @@
+  background-color: #f3f3fa;
+}
+
+header {
+  background-color: #fff;
+  height: 60px;
+}
+
+header nav {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+  margin: 0 auto;
+  padding: 1px 10px;
+}
```
- [ ] [Optional] [Highly recommended] Kindly there is a margin that appears on the right side of the nave bar on both home and about pages. Please fix that (See the image in the comment above).

- [ ] [Optional] [Highly recommended] In the desktop view please center the logo and nav links to make your site more beautiful and match the given design.

![image](https://user-images.githubusercontent.com/68796232/169312440-14aeaebb-fb16-4f25-a050-14a61468b187.png)

Omar-Muhamad reviewed (approved):
# Approved ✔
### Hi @Josphat205,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### ⚠️Please consider the Optional changes below because it's very important and I approved PR changes and marked them as optional because the last reviewer forgot to mention it. ⚠️

![](https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg)

## (Highlights) Good Points: :+1:
- Your design looks beautiful and professional. ✔
- Nice work with the animation. ✔
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- Please check the inline comments below ⬇.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Josphat205 commented:
### Changes made 

- Aligned items center on the navbar
